.env.dist

```diff
@@ -1,10 +1,12 @@
 # .env.test
-DB_HOST=127.0.0.1
```

Comment:
I think these changes broke travis build. Did you remove the default credentials on purpose?